devops/openshift/api-deploy.yml

```diff
@@ -350,6 +352,8 @@ parameters:
     value: FORMS_URL
   - name: SITE_MINDER_LOGOUT_URL
     require: true
+  - name: IS_FULLTIME_ALLOWED
+    require: true
```

Comment:
According to openshift documentation this must be `required` not `require`
@andrewsignori-aot @guru-aot , I see we are using 2 different properties (yml key) `require` and `required`

![image](https://github.com/bcgov/SIMS/assets/54600590/41c24d31-7002-4932-9b60-b2df0faa9b5a)

![image](https://github.com/bcgov/SIMS/assets/54600590/4109d826-3018-4f32-80e7-111e24987a1d)